sdk/messaging/azservicebus/ci.yml

```
@@ -0,0 +1,17 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/messaging/azservicebus
+
+pr:
+  paths:
+    include:
+    - sdk/messaging/azservicebus
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'messaging'
```

```suggestion
    ServiceDirectory: 'messaging/azservicebus'
```

For the go repo the ServiceDirectory needs to be the full relative path from sdk folder.

---

Done: https://github.com/Azure/azure-sdk-for-go/pull/15699/commits/89248b29071750ef84b135a518a85965640d2a2f